config/samples/simple.yaml

```diff
@@ -33,7 +33,8 @@ spec:
   file_storage_storage_class: "standard"
   signing_secret: "signing-galaxy"
   signing_scripts_configmap: "signing-scripts"
-  image_web: "quay.io/pulp/galaxy-web"
+  image_web: "quay.io/pulp/pulp-web"
+  #image_web: "quay.io/pulp/galaxy-web"
```

Comment:
why not delete it?

Reply:
My bad!
I was doing some tests and should not have even pushed this.